### PR TITLE
Load & retrieve medicine names from DynamoDB

### PR DESCRIPTION
**Pull request by SaraPrager**
This is the first attempt: to load & retrieve the data from Dynamo.
I don't like this direction, it became too comlicated. Checking other solutions (maybe AWS OpenSearch?)

### CODE REVIEW

File: infra/dynamo-db.yml
Comment (dmichaelb) [inline infra/dynamo-db.yml:1]: Nice !

File: src/types/medicine.ts [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
-    medicineId: string,
-    names: string[]
+export interface MedicineName {
+    medicine_id: string,
+    name: string,
+    name_in_second_lang: string
+};
+
+export function dbItems2MedicineNames(items: any[]): MedicineName[] {
+    return items.map(({ medicine_id, name, name_in_second_lang }) => ({ medicine_id, name, name_in_second_lang }));
```
Comment (dmichaelb) [inline src/types/medicine.ts:8]: not sure I understand the purpose

File: src/utils/excel-utils.ts
Comment (dmichaelb) [inline src/utils/excel-utils.ts:1]: goo to have !

File: src/utils/logger.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ if (!config.IS_CLOUD) {
 }
 
 export const logger = winston.createLogger({
-  level: config.isDevelopment ? 'debug' : 'info',
+  level: config.isDevelopment ? 'debug' :  process.env.LOG_LEVEL || 'info',
```
Comment (dmichaelb) [inline src/utils/logger.ts:12]: nice

File: src/services/dynamo-service.ts
Comment (dmichaelb) [inline src/services/dynamo-service.ts:1]: good usage of dynamo service

**Review by dmichaelb — APPROVED** [review]
LGTM ! 
Not sure I understand what's complicated and why Opensearch will solve it

File: src/types/medicine.ts (hunk 1 above)
Comment (SaraPrager) [reply src/types/medicine.ts:8]: To omit irrelevant fields from response

**Pull request merged**